rules/predicates.py

```diff
@@ -211,9 +216,12 @@ def _apply(self, *args):
             callargs = (self,) + callargs
         try:
             result = self.fn(*callargs)
-            return None if result is None else bool(result)
+            returner = None if result is None else bool(result)
```

Comment:
It's safe to overwrite `result`. "returner" feels somewhat awkward as an identifier here.